sub-packages/bionemo-esm2/src/bionemo/esm2/model/finetune/train.py

```diff
@@ -166,24 +170,20 @@ def train_model(
     dataset = InMemorySingleValueDataset(data)
     data_module = ESM2FineTuneDataModule(train_dataset=dataset, valid_dataset=dataset)
 
-    with tempfile.TemporaryDirectory() as experiment_tempdir_name:
-        experiment_dir = Path(experiment_tempdir_name)
-        experiment_name = "finetune_regressor"
-        n_steps_train = 50
-        seed = 42
-
-        # To download a pre-trained ESM2 model that works with this inference script, run the following command...
-        # $ download_bionemo_data esm2/650m:2.0 --source ngc
-        # ... and pass the output path (e.g. `.../.cache/bionemo/975d29ee980fcb08c97401bbdfdcf8ce-esm2_650M_nemo2.tar.gz.untar`)
-        # as an argument into `initial_ckpt_path` below!
-        config = ESM2FineTuneSeqConfig(
-            # initial_ckpt_path=str(pretrain_ckpt_path)
-        )
+    experiment_name = "finetune_regressor"
+    n_steps_train = 50
+    seed = 42
 
-        checkpoint, metrics, trainer = train_model(
-            experiment_name=experiment_name,
-            experiment_dir=experiment_dir,  # new checkpoint will land in a subdir of this
-            config=config,  # same config as before since we are just continuing training
-            data_module=data_module,
-            n_steps_train=n_steps_train,
-        )
+    # To download a 650M pre-trained ESM2 model
+    pretrain_ckpt_path = load("esm2/650m:2.0", source="ngc")
+
+    config = ESM2FineTuneSeqConfig(initial_ckpt_path=str(pretrain_ckpt_path))
+
+    checkpoint, metrics, trainer = train_model(
+        experiment_name=experiment_name,
+        experiment_dir=Path(experiment_results_dir),  # new checkpoint will land in a subdir of this
+        config=config,  # same config as before since we are just continuing training
+        data_module=data_module,
+        n_steps_train=n_steps_train,
+    )
+    print(f"Experiment completed with checkpoint stored at {checkpoint}")
```

Comment:
One idea, if you print these two messages as two prints, then the last output line of the script will be the checkpoint. You can use that to further automate your notebooks/examples. For example:
```
print("Experiment completed with checkpoint stored at ")
print(f"{checkpoint}")
```
Then in your notebook you can have something like:
```
train_out:list[str] = !python -m bionemo.esm2.model.finetune.train [args] [whatever]
ckpt_path = train_out[-1]
```

Reply:
I am going to try using the pretrain script to run finetuning and deprecate this script in another PR

Reply:
great, either way printing the checkpoint as the last step would get you this ability.

Reply:
so maybe we should do that in `train.py` in general?

Reply:
yes, we definitely need that for finetuning, I will add that to pre-train script